### DISCUSSION TITLE
Mysql or Oracle instead Cassandra

### DISCUSSION THREAD

**Discussion by kisjaj2** [glowroot/glowroot#1015] [General]
Dear All,

I would like to get information about usage another DB solution as Cassandra.  Is there any way to use MYSQL or Oracle as DB engine for Glowroot Central?

If yes, where can I find any documentation for setup this configuration? 

Hope you can help.  

Regards,
Kisjaj

**Comment by nowheresly** [comment]
Hi

so far only cassandra is supported for glowroot central. (I guess maybe scylladb could work also, yet I never tested it).
